.golangci.yml

```diff
@@ -0,0 +1,20 @@
+---
+linters-settings:
+  lll:
+    line-length: 140
+    tab-width: 4
+
+  unused:
+    check-exported: true
```

Comment:
Wouldn't this generate too many false positives?

Reply:
shouldn't, so it's config for all linters including optional ones.